templates/etc/prometheus/prometheus.yml.j2

```diff
@@ -10,6 +10,7 @@ rule_files:
   - {{ prometheus_rules_dir }}/{{ dest }}
   {% endfor %}
 {% endif %}
+  - /etc/prometheus/non-ansible-managed/rules/*.rules
```

Comment:
Shouldn't you change this to include the `{{ prometheus_non_ansible_managed }}` variable instead of hard-coding `non-ansible-managed` ?

Reply:
Yes indeed :) Unfortunately I noticed the same only after merging this PR already. I'll push a fix later.